docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -69,6 +69,28 @@ If no conditions met for a data part, ClickHouse uses the `lz4` compression.
 </compression>
 ```
 
+## encryption {#server-settings-encryption}
+
+Configures a command to obtain a key to be used by [encryption codecs](../../sql-reference/statements/create/table.md#create-query-encryption-codecs). The command, or a shell script, is expected to write a Base64-encoded key of any length to the stdout.
+
+**Example**
+
+For Linux with systemd:
+
+```xml
+<encryption>
```

Comment:
This configuration can be confused with the configuration of entrypted disks.

Reply:
Yes, a longer path in xml would be better, for example

<codecs>
    <aes_128_gcm_siv>
        <key_hex>...</key_hex>
    </aes_128_gcm_siv>
</codecs>